doc/source/templates/03_serving_stable_diffusion/README.md

```diff
@@ -5,7 +5,7 @@
 | Summary | This app provides users a one click production option for serving a pre-trained Stable Diffusion model from HuggingFace.  It leverages [Ray Serve](https://docs.ray.io/en/latest/serve/index.html) to deploy locally and built in IDE integration on an Anyscale Workspace to iterate and add additional logic to the application. You can then use a simple CLI to deploy to production with [Anyscale Services](https://docs.anyscale.com/productionize/services/get-started). |
 | Time to Run | Around 2 minutes to setup the models and generate your first image(s). Less than 10 seconds for every subsequent round of image generation (depending on the image size). |
 | Minimum Compute Requirements | At least 1 GPU node. The default is 4 nodes, each with 1 NVIDIA T4 GPU. |
-| Cluster Environment | This template uses a docker image built on top of the latest Anyscale-provided Ray image using Python 3.9: [`anyscale/ray:latest-py39-cu118`](https://docs.anyscale.com/reference/base-images/overview). See the appendix below for more details. |
+| Cluster Environment | This template uses a docker image built on top of the latest Anyscale-provided Ray 2.9 image using Python 3.9: [`anyscale/ray:latest-py39-cu118`](https://docs.anyscale.com/reference/base-images/overview). See the appendix below for more details. |
```

Comment:
Why do we use the py39-cu118 vs anyscale/ray-ml:2.9.0-py39-gpu?

Reply:
I'll try it out, I believe it was first set up this way because ray-ml has some conflicting dependencies? @matthewdeng @justinvyu ?

Reply:
I used the anyscale/ray-ml:2.9.0-py39-gpu and installed diffusers, transformers, and accelerate on all nodes. It worked fine when generating images.